packages/webpack-cli/package.json

```diff
@@ -59,5 +59,8 @@
       "optional": true
     }
   },
-  "gitHead": "fb50f766851f500ca12867a2aa9de81fa6e368f9"
+  "gitHead": "fb50f766851f500ca12867a2aa9de81fa6e368f9",
+  "devDependencies": {
+    "webpack-dev-server": "^4.7.4"
+  }
```

Comment:
I've added `webpack-dev-server` as dev dep in `webpack-cli` package, but the smoketests are still failing when resolving `webpack-dev-server` in `serve` package. [(check the details here)](https://github.com/webpack/webpack-cli/runs/5214548107?check_suite_focus=true) 

Should I add `webpack-dev-server` as a peerDependency in `serve` package? 🤔

Reply:
In theory yes, but there is the problem, `serve` is part of `webpack-cli`, so if we add `webpack-dev-server` to `peerDependency` new npm verison will install it automatically, it is not good, because many projects don't use dev server, I think better refactor our tests

Reply:
Thank you for the explanation. I managed to reproduce the CI error locally and fix it by removing the dev dep from `webpack-cli` and surpressing extraneous import in TS and eslint.